src/features/bivariate_manager/components/BivariateMatrixControl/components/QuotientSelector/QuotientSelector.tsx

```diff
@@ -18,112 +19,47 @@ const QuotientItem = ({
   numeratorLabel,
   quality,
   onSelectQuotient,
+  isSelected = false,
 }: QuotientItemProps) => {
   const onClick = (ev) => {
     ev.stopPropagation();
     onSelectQuotient(numeratorId, denominatorId);
   };
 
   return (
-    <div onClick={onClick}>
+    <div className={cn(isSelected && styles.selected)} onClick={onClick}>
       <DenominatorIcon iconId={denominatorId} />
-      <div className="qualityLabel">
+
+      {/* Do not remove, stay for testing purposes */}
```

Comment:
maybe we want to turn it on automatically in dev mode?

Reply:
nope, I was asked to stay it for qa for case if something happens and we need to debug it

Reply:
How qa can enable it in that case?

Reply:
ask me to uncomment and deploy to dev

Reply:
oh, I like it! thanks for the idea!